nethttp/client.go

```diff
@@ -64,6 +65,14 @@ func ClientTrace(enabled bool) ClientOption {
 	}
 }
 
+// InjectSpanContext returns a ClientOption that turns on or off
+// injection of the Span context in the request HTTP headers.
```

Comment:
add: If this option is not used, the default behavior is to inject span context.